README.org

```diff
@@ -11,6 +11,10 @@ Should I use ScalaZ or Cats?
 
 The answer is, of course, "it depends". What's your use-case?
 
+*I am concerned about developer knowledge*
```

Comment:
Does "developer knowledge" refer to the devs being trained, or the devs who wrote the libraries? I don't think I can make claims about the latter. Do you think the following might be clearer?

> I am concerned about training my Scala team in the fundamentals.

Reply:
your version reads better :smile:

Reply:
Then for sure I agree that having a book to work through together would be like night-and-day. My company is currently getting around this by working through The Haskell Book (small team, opt-in only), hoping that the knowledge there will trickle down into other Scala / JS work.